### PR TITLE
[Listing Service] Define entities using TypeORM

### PR DESCRIPTION
**Pull request by mitidevus**
- Define Hotel entity using TypeORM
- Task: [Link](https://github.com/orgs/nashtech-garage/projects/9/views/1?pane=issue&itemId=79039962)

### CODE REVIEW

File: backend/nestjs/apps/listing-service/src/hotels/hotels.controller.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import {
```
Comment (khanhprolazay) [inline backend/nestjs/apps/listing-service/src/hotels/hotels.controller.ts:1]: Controller must listen to message broker (Kafka) and react to correspond event and doesn't expose apis, following on the architecture

File: backend/nestjs/apps/listing-service/src/database/database.module.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Module } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+@Module({
+  imports: [
+    TypeOrmModule.forRootAsync({
+      useFactory: (configService: ConfigService) => ({
+        type: 'postgres',
+        host: configService.getOrThrow('POSTGRES_HOST'),
```
Comment (tuonglevan) [inline backend/nestjs/apps/listing-service/src/database/database.module.ts:10]: Add prefix `LISTING_` for all env about listing service

**Comment by tuonglevan** [comment]
@khanhprolazay 
1. What does migration and seeding define ?
2. Should be add some environment into docker-compose.yml

File: docker-compose.yml [hunk 3]
```diff
@@ -154,6 +154,11 @@ services:
       dockerfile: ./apps/listing-service/Dockerfile
     image: nodejs-msa-listing:dev
     container_name: listing
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
```
Comment (tuonglevan) [inline docker-compose.yml:158]: Should be Priorities 
- POSTGRES_HOST
- POSTGRES_PORT
- POSTGRES_USER
- POSTGRES_PASSWORD

**Review by tuonglevan — APPROVED** [review]
Reviewed & approved without migrations scripts

**Pull request merged**